README.rst

```diff
@@ -389,6 +389,24 @@ where ``${TARGET}`` is one of the following
 
 .. docker-end-inclusion-marker-do-not-remove
 
+LAPACK support
+**************
+
+LAPACK, a numerical linear algebra library, is required to enable stochastic measurement support in the C++ backend. We suggest first installing ``LAPACK`` following instructions in `LAPACK document <https://github.com/Reference-LAPACK/lapack/blob/db501d91ba690192dfe946524251b02d416298d0/README.md?plain=1#L85-L92>`_.
+
+On Debian-based Linux systems, LAPACK can be also installed via ``apt``:
+
+.. code-block:: console
+
+    $ sudo apt -y update &&
+    $ sudo apt install g++ liblapack-dev
+
+where ``liblapack-dev`` is included to also install LAPACK.
+
+On Windows systems, LAPACK is recommended to be built and installed using `vcpkg <https://github.com/Microsoft/vcpkg/>`_ following the instructions `here <https://github.com/Reference-LAPACK/lapack/blob/db501d91ba690192dfe946524251b02d416298d0/README.md?plain=1#L85-L92>`_.
```

Comment:
please add a brief description to install LAPACK on macOS.

Reply:
Done!